src/docs/advanced/use-sealer-rootless.md

```diff
@@ -0,0 +1,27 @@
+# Run sealer in non-root mode
+
+## Install sealer using sudo
+
+**Resolved remote read permission to `sshd_config` file:**
+
+```bash
+$ sudo chmod 644 /etc/ssh/sshd_config
+```
+
+
+
+### Ensure all nodes participating in the cluster have passwordless login permissions
+
+**Still needs to assign non-secret permissions to user sudo(Every node except master0):**
+
+You need to give passwordless login permissions to users on all nodes: `vim /etc/sudoers`. It is a configuration file that contains sudo user and group rules, allowing a specific user (sealer) or group to run commands as a superuser when executing commands.
+
+```bash
+sealer ALL=(ALL) NOPASSWD: NOPASSWD: ALL
```

Comment:
is this the rules that we need to make sure it is meet our expectation ? we could describe it more clearly .

Reply:
yes, but this one seems very mature and I think very detailed